test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -23,13 +23,51 @@
 using namespace std;
 using namespace yul::test::yul_fuzzer;
 
+std::string yul::test::yul_fuzzer::createHex(std::string const& _hexBytes)
+{
+	std::string tmp{_hexBytes};
+	if (!tmp.empty())
+	{
+		tmp.erase(std::remove_if(tmp.begin(), tmp.end(), [](char c) {
+			return !std::isxdigit(c);
+		}), tmp.end());
+		tmp = tmp.substr(0, 64);
+	}
+	// We need this awkward if case because tmp can also be null post erase call above
+	if (tmp.empty())
+		tmp = "1";
+	return tmp;
+}
+
+std::string yul::test::yul_fuzzer::createAlphaNum(std::string const& _strBytes)
+{
+	std::string tmp{_strBytes};
+	if (!tmp.empty())
+	{
+		tmp.erase(std::remove_if(tmp.begin(), tmp.end(), [](char c) {
+			return !(std::isalpha(c) || std::isdigit(c));
+		}), tmp.end());
+		tmp = tmp.substr(0, 32);
+	}
+	// We need this awkward if case because tmp can also be null post erase call above
```

Comment:
Actually - wouldn't the empty string be a valid literal?

Reply:
Returning empty string increases the number of parsing errors because code like this is invalid

```
switch x
case "" {}
```

Reply:
I'm wrong. These are parsed successfully.

```
{
        let x := 1
        if ""
        {
        }
}
```

and

```
{
        let x := 1
        switch x                           
        case "" {
        }
}
```